analytics/analytics.properties

```diff
@@ -13,3 +13,7 @@ instance=geOrchestra
 # Timezone to convert datetime sent by UI to UTC
 # (see Canonical ID on http://joda-time.sourceforge.net/timezones.html for possible values)
 localTimezone=Europe/Paris
+
+# List of user to exclude in statistics
+protectedUser.uid1=geoserver_privileged_user
+protectedUser.uid2=anonymousUser
```

Comment:
I would remove anonymousUser by default since it is useful IMO for the common case, eg:

![capture du 2017-01-19 12-59-45](https://cloud.githubusercontent.com/assets/265319/22106031/2a6af094-de47-11e6-9f4c-d73a20a3a7c7.png)